omero/sysadmins/customization.txt

```diff
@@ -0,0 +1,119 @@
+Customize your public repository
+================================
+
+The OMERO clients offer flexible user interface that can be customized.
+The sections below describe how to set up these features.
+
+Index Page
+----------
+
+Create new custom template in /your/path/to/templates/mytemplate/index.html
+and add the following
+
+.. note::
+    User will no longer be automaticaly redirected to the login page
+
+::
+
+    $ bin/omero config set omero.web.template_dirs '/your/path/to/templates/'
+    $ bin/omero config set omero.web.index_template 'mytemplate/index.html'
+
+.. figure:: /images/indexPage.png
+
+
+Login Page
+----------
+
+:property:`omero.web.login_logo` allows to customize webclient login page
+with your own logo. Logo images should ideally be 150 pixels high or less
+and will appear above the OMERO logo. You will need to host the image
+somewhere else and link to it with 
+
+::
+
+    $ bin/omero config set omero.web.login_logo 'http://www.url/to/image.png'
+
+.. figure:: /images/customLogin.png
+
+
+Login redirection
+-----------------
+
+:property:`omero.web.login_redirect` property redirects to the givin location
+after loging in.
+
+::
+
+    $ bin/omero config set omero.web.login_redirect '{"redirect": ["webindex"], "viewname": "load_template", "args":["userdata"], "query_string": "experimenter=-1"}'
+
+
+
+Top Links Menu
+--------------
+
+:property:`omero.web.ui.top_links` adds links to the top header.
+
+::
+
+    $ bin/omero config append omero.web.ui.top_links '["Figure", "webfigure"]'
+    $ bin/omero config set omero.web.ui.top_links '["GRE", "http://lifesci.dundee.ac.uk/gre"]'
+
+.. figure:: /images/topLink.png
+
+
+Group and Users in dropdown menu
+--------------------------------
+
+Customize group and users drop down menu by changing labels or hiding entire
```

Comment:
Customize **the** group**s** and users dropdown menu by changing **the** labels or hiding **the** entire list.